### ISSUE TITLE
Support Seattle's Tier 2 languages throughout CiviForm

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#977]
[Here's the list](https://www.seattle.gov/iandraffairs/LA): 

> Tigrinya, Oromo, Japanese, Russian

**Comment by elisekalstad** [comment]
What would it take to add Oromo, Japanese, Tigrinya, and Russian? The Ride Now program would like to onboard to CiviForm, but residents/CBOs requested that it include those Tier 2 languages.

**Comment by bion** [comment]
@elisekalstad can Seattle help with translations?

**Comment by elisekalstad** [comment]
> @elisekalstad can Seattle help with translations?

Yes of course! I just mean, what would it take to build those language templates within the tool? Right now I can copy/paste translations for the 7 languages. I just need the additional tabs for Oromo, Japanese, Tigrinya, and Russia.

**Comment by bion** [comment]
Ah got it, cc @gwendolyngoetz 

To add a new language we add a messages file like this one: https://github.com/seattle-uat/civiform/blob/deployment-doctor/universal-application-tool-0.0.1/conf/messages.am

And add the language to the list of supported languages here: https://github.com/seattle-uat/civiform/blob/deployment-doctor/universal-application-tool-0.0.1/conf/application.conf#L163

jfyi we currently have some missing messages for languages we already support, and those can be found using `bin/list-incomplete-messages`

**Comment by gwendolyngoetz** [comment]
Looks pretty straightforward. I'll work with @elisekalstad to get translations for the four new message files and then wire it up.

**Comment by gwendolyngoetz** [comment]
I have a branch all set. Just need to copy over the translations when we get them.

**Comment by elisekalstad** [comment]
Recommendation from OIRA: use data collected in CiviForm on preferred language to determine the next languages to onboard. OIRA recently removed the "tier" system and instead lists the top 15 languages spoken in Seattle. Supporting 15 languages may not be feasible, so we should consider which languages should be prioritized based on current utilization or gaps in community outreach. @Miguel-314  for awareness.

**Comment by gwendolyngoetz** [comment]
@elisekalstad Can you put the updated list of new languages you want added in here?

**Comment by gwendolyngoetz** [comment]
Arabic will have to wait due to the right-to-left language support needed to be added to the application, but the others shouldn't be a problem

**Comment by gwendolyngoetz** [comment]
@shreyachatterjee00 FYI

**Comment by dkatzz** [comment]
FYI I put in a request for the general program text for Russian and Japanese.

**Comment by rockycodes** [comment]
@elisekalstad which languages are still left to do on this issue?

**Comment by gwendolyngoetz** [comment]
@rockycodes I don't know if it has changed given how we use Transifex now, but in the past we were not able to get translations for Oromo and Tigrinya according to Danielle.

**Comment by rockycodes** [comment]
Darn, it seems like the Transifex vendors we're using now also do not translate into those languages.

**Comment by gwendolyngoetz** [comment]
Ok so its the same as before when Google was doing it.

**Comment by gwendolyngoetz** [comment]
> Arabic will have to wait due to the right-to-left language support needed to be added to the application, but the others shouldn't be a problem

This is no longer an issue on the applicant side. We've only tested with Arabic, but any right-to-left language should work.
